tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/ClientCredentialsTests.cs

```diff
@@ -75,8 +75,8 @@ public async Task WithSecret_TestAsync(Cloud cloud, RunOn runOn)
         }
 
         [TestMethod]
-        [DataRow(Cloud.Public, RunOn.NetFx | RunOn.NetCore)]
-        [DataRow(Cloud.Adfs, RunOn.NetFx)]
+        //[DataRow(Cloud.Public, RunOn.NetFx | RunOn.NetCore)]
```

Comment:
>//[DataRow(Cloud.Public, RunOn.NetFx | RunOn.NetCore)] [](http://example.com/codeflow?start=8&length=54)

why not on public?

Reply:
Accidental commit of this file, I will revert it.